units/systemd-journal-gatewayd.service.in

```diff
@@ -20,6 +20,11 @@ PrivateDevices=yes
 PrivateNetwork=yes
 ProtectSystem=full
 ProtectHome=yes
+ProtectControlGroups=yes
+ProtectKernelTunables=yes
+MemoryDenyWriteExecute=yes
+RestrictRealtime=yes
+RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
```

Comment:
the order in which selinux and seccomp (and thus nnp) are applied is something we should figure out in a different PR, see #3993.